apps/mobile/tailwind.config.js

```diff
@@ -6,7 +6,11 @@ module.exports = {
     './app/**/*.{js,jsx,ts,tsx}',
   ],
   theme: {
-    extend: {},
+    extend: {
+      height: {
+        25: 104
```

Comment:
@rodrigorgtic aqui o correto seria `26` já que `104 / 4 = 26` e provavelmente usar REM ao invés de `px`:
```suggestion
        26: '6.5rem'
```